autoload/neomake/makers/mvn.vim

```diff
@@ -4,6 +4,29 @@ function! neomake#makers#mvn#mvn()
     return {
          \ 'exe': 'mvn',
          \ 'args': ['install'],
-            \ 'errorformat': '[%tRROR]\ %f:[%l]\ %m,%-G%.%#'
+         \ 'remove_invalid_entries' : 1,
+         \ 'postprocess': function('neomake#makers#mvn#InvalidDuplicates'),
+         \ 'errorformat':
+           \ '%E[%tRROR]\ %f:[%l]\ %m,' .
+           \ '%E[%tRROR]\ %f:[%l\,%v]\ %m,' .
```

Comment:
Just wondering, why %v and not %c? I think that maven-compiler-plugin is reporting tabs as single column, or am I wrong?